### PR TITLE
TDI-37879:tMySqlOutput generates invalid create statement when DOUBLE

### PR DESCRIPTION
**Pull request by pyzhou-talend**
https://jira.talendforge.org/browse/TDI-37879

### CODE REVIEW

**Comment by pyzhou-talend** [comment]
I have done some test and does not find any harmful influence.

File: main/plugins/org.talend.designer.components.localprovider/components/templates/db_output_bulk.skeleton [hunk 1]
```diff
@@ -383,7 +383,7 @@ public class CLASS {
                                 && (column.getColumn().getPrecision() == null || 0 == column.getColumn().getPrecision())
                         ){} 
                         else if (("mysql_id".equalsIgnoreCase(getDBMSId()))
-                                && (("DECIMAL".equalsIgnoreCase(dataType)) || ("NUMERIC".equalsIgnoreCase(dataType)))
+                                && (("DECIMAL".equalsIgnoreCase(dataType)) || ("NUMERIC".equalsIgnoreCase(dataType))||(("FLOAT".equalsIgnoreCase(dataType)))||("DOUBLE".equalsIgnoreCase(dataType)))
```
Comment (jzhao-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/templates/db_output_bulk.skeleton]: I think only this one is for mysql.

File: main/plugins/org.talend.designer.components.localprovider/components/templates/db_output_bulk.skeleton [hunk 2]
```diff
@@ -1501,7 +1501,7 @@ public class CLASS {
                                 && (column.getColumn().getPrecision() == null || 0 == column.getColumn().getPrecision())
                         ){} 
                         else if (("mysql_id".equalsIgnoreCase(getDBMSId()))
```
Comment (jzhao-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/templates/db_output_bulk.skeleton]: Useless code. Should consider to remove this kind of code

File: main/plugins/org.talend.designer.components.localprovider/components/templates/db_output_bulk.skeleton [hunk 3]
```diff
@@ -3591,7 +3591,7 @@ public class CLASS {
                                 && (column.getColumn().getPrecision() == null || 0 == column.getColumn().getPrecision())
```
Comment (jzhao-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/templates/db_output_bulk.skeleton]: Same as above

**Pull request merged**